requirements.txt

```diff
@@ -1,6 +1,7 @@
 google_api_python_client==1.6.4
 google_auth_httplib2==0.0.2
 gunicorn==20.0.4
+jellyfish==0.9.0
```

Comment:
there is a method in standard python `difflib` to get a similarity of 2 strings, but after brief investigation it seems  Levenshtein algorithm pefroms better and faster.